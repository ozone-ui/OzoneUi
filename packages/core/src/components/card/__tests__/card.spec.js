import { describe, it, expect } from 'vitest'
import { shallowMount, mount } from '@vue/test-utils'
import { Card } from './'
let wrapper

describe('Card', () => {
  beforeEach(() => {
    wrapper = mount(Card, {
      props: {
        title: 'The Settings',
      },
    })
  })

  it('is called', () => {
    expect(wrapper.exists()).toBeTruthy()
  })
})

it('matches snapshot', () => {
  expect(wrapper.html()).toMatchSnapshot()
})

// props title with class card-title
it('has title', () => {
  expect(wrapper.html()).toContain('The Settings')
})
