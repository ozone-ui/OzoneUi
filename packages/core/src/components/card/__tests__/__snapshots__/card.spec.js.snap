// Vitest Snapshot v1

exports[`matches snapshot 1`] = `
"<div class=\\"px-[20px] py-[30px] rounded-[15px] w-full h-fit flex flex-col card-title\\" style=\\"border-color: transparent; background: rgb(247, 247, 247);\\">
  <h1 class=\\"text-[20px] font-bold text-black\\">The Settings</h1>
  <!---->
</div>"
`;
